.claude/skills/pr-flow/SKILL.md

```
@@ -1,350 +1,212 @@
+gh api graphql -F threadId="<THREAD_ID>" -f query='
+mutation($threadId: ID!) {
+  resolveReviewThread(input: { threadId: $threadId }) {
+    thread { id isResolved }
+  }
+}'
 ```
 
-### Step 4: Parse and Classify CodeRabbit Feedback
-
+#### Verify all threads resolved:
 ```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)